blocks/priority-list/priority-list.css

```diff
@@ -443,10 +451,13 @@ html[dir="rtl"] .priority-list-wrapper .priority-list .right-column-container .c
     .priority-list
     .left-column-container
     .image-container {
+    max-width: 731px;
+    max-height: 731px;
```

Comment:
Will this restrict the content from growing?

Reply:
It is to restrict the image from growing . will not affect content on right side